roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1066,36 +1066,40 @@ def get_openshift_version(facts, cli_image=None):
         if 'version' in facts['common'] and facts['common']['version'] is not None:
             return facts['common']['version']
 
-    if os.path.isfile('/usr/bin/openshift'):
-        _, output, _ = module.run_command(['/usr/bin/openshift', 'version'])
-        version = parse_openshift_version(output)
-
+    # If this is a containerized host, we need to get the openshift version
```

Comment:
Yeah looks like a lot of overlap. I'm going to go test running without docker pre-installed on @brenton 's PR and I'll send over a patch if anything is still missing there.